### PR TITLE
Fix missing "and" warnings

### PR DESCRIPTION
**Pull request by Efnilite**
### Description
Fixes missing `and` warnings in a few tests. This reduces the amount warnings that show up, increasing the chances of contributors finding warnings in their own PRs, and making it easier to find errors.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

**Review by sovdeeth — APPROVED** [review]

File: src/test/skript/tests/regressions/pull-6586-effhealth item mutation.sk [hunk 1]
```diff
@@ -9,6 +9,6 @@ test "EffHealth item mutation fix":
 
 	set {_item1} to diamond sword with damage 100
 	set {_item2} to diamond with damage 10
-	repair {_item1}, {_item2}
+	repair {_item1}, and {_item2}
```
Comment (APickledWalrus) [inline src/test/skript/tests/regressions/pull-6586-effhealth item mutation.sk:12]: Can we remove the comma in cases like this where there are only two values?

**Review by APickledWalrus — CHANGES_REQUESTED** [review]

**Review by APickledWalrus — APPROVED** [review]

**Comment by APickledWalrus** [comment]
Oh this could probably target patch if you want to rebase it

**Comment by Moderocky** [comment]
Seems it didn't like changing the base, I can rebase this for you if you need.

**Comment by Moderocky** [comment]
I think I fixed it for you please double check it looks right though!

**Comment by Efnilite** [comment]
SecFilter and ExprJoinSplit are not up to date, do I have to redo them

**Comment by Moderocky** [comment]
> SecFilter and ExprJoinSplit are not up to date, do I have to redo them

Don't worry I removed them (they were feature-only tests) it should be all fine now.

**Pull request merged**